content/Teams/reads/index.md

```
@@ -0,0 +1,154 @@
+- **Eden Uzoma** — AI Engineer & Frontend Developer
+- **Borngreat Ikwutah** — Smart Contracts Developer
+- **Maruh Akporowho** — Backend Developer
+- **David Ignatius** — Backend Developer
+- **Ugochukwu David Anya** — Frontend Developer
+- **Ibrahim Danjuma** — Frontend Developer
+- **Akinwande Kehinde (Ebenezer)** — UI/UX Designer
+
+---
+
+## Pages
+## Pages
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)